lib/wordpress.js

```diff
@@ -44,7 +44,9 @@ function Client( settings ) {
 		// via settings.host
 		headers: {
 			Host: parsedUrl.host
-		}
+		},
+
+		basic_auth: settings.basic_auth
```

Comment:
to avoid the name mismatch I would suggest using

```js
// ....
	this.rpc = xmlrpc[ parsedUrl.secure ? "createSecureClient" : "createClient" ]({
		host: settings.host || parsedUrl.host,
		port: parsedUrl.port,
		path: parsedUrl.path,

		// Always set Host header in case we're pointing to a different server
		// via settings.host
		headers: {
			Host: parsedUrl.host
		},
        basic_auth: settings.basicAuth ? {
            user: settings.basicAuth.username,
            pass: settings.basicAuth.password
        } : null,
	});
// ....
```